### DISCUSSION TITLE
Compiled Windows Executable MPC.EXE mpc-0.33 Requested

### DISCUSSION THREAD

**Discussion by Maxol** [MusicPlayerDaemon/MPD#1246] [Q&A]
Recently I've found MPD and I'm very impressed! I'd like to dig deeper into the mechanism of the program and MPC seems to be an excellent starting point. Unfortunately I'm using an almost 10 year old version, so can anyone provide me with a compiled MPC.EXE for Windows version mpc-0.33?

I've looked in a lot of places but I only managed to find version 0.22 inside the zipfile on [https://www.musicpd.org/download/mpc/0/](url)

**Comment by bobrathbone** [comment]
What platform are you running MPD on?  If it the Raspberry Pi see the following web page.
[https://www.bobrathbone.com/raspberrypi/compiling_mpd.html](url)
It isn't  the latest version of **mpc** you need but MPD.
The current version on the above site is 0.22.9 but I will shortly be upgrading this to 0.22.11
It could be possibly adapted to other platforms.
BTW your link above isn't working.
Bob Rathbone

**Reply by Maxol** [reply]
I'm running both MPD and MPC on Windows7 so I'm looking for a compiled Windows Executable (mpc-0.33). I don't have the tools to compile it so the sources don't work for me. 
Thank you!

**Reply by bobrathbone** [reply]
Sorry but I am unable to help as I only work with MPD on Debian Linux.
  Good luck finding a solution.

**Comment by bobrathbone** [comment]
It looks like this forum is resolving all links to https://github.com/MusicPlayerDaemon/MPD/discussions/url instead of the published link.
The following link should work if not copy and paste into your browser
https://www.bobrathbone.com/raspberrypi/compiling_mpd.html 
Bob Rathbone

**Comment by MaxKellermann** [comment]
Sorry for not replying for so long - I just didn't have the time to take care for mpc yet. But I'll include Windows binaries in the next release.

**Reply by Maxol** [reply]
That would be great! I'll keep an eye on the releases...
I'm not in a hurry so relax ;-)

**Comment by MaxKellermann** [comment]
`mpc.exe` is now built on every push by GitHub actions: https://github.com/MusicPlayerDaemon/mpc/actions/workflows/build.yml
Click on a successful build, and you'll see a download link.

**Reply by Maxol** [reply]
> `mpc.exe` is now built on every push by GitHub actions: https://github.com/MusicPlayerDaemon/mpc/actions/workflows/build.yml Click on a successful build, and you'll see a download link.

I've been studying this 'push' GitHub puzzle but I cannot find any download link for Windows. I see 'Artifact mpc.exe has been successfully uploaded!' in line 9 of step 'Upload mpc.exe' but where can I find it?

**Reply by naglis** [reply]
@Maxol please check [Downloading workflow artifacts](https://docs.github.com/en/actions/managing-workflow-runs/downloading-workflow-artifacts) in _GitHub_ documentation. E.g. for [this run](https://github.com/MusicPlayerDaemon/mpc/actions/runs/1522178436) the link is: https://github.com/MusicPlayerDaemon/mpc/suites/4509635710/artifacts/120463419 (the _Artifacts_ section is at the bottom of the page).

**Reply by Maxol** [reply]
Thank you naglis, appreciate it!

**Reply by lxg208** [reply]
Looks like the artifact expired.

**Comment by mc510** [comment]
Is mpc.exe for win32 still intended to be built automatically? It appears to have failed on recent push actions, and is "expired" on older actions.
